src/auto-evo/RunResults.cs

```diff
@@ -368,6 +389,129 @@ public long GetPopulationInPatch(Species species, Patch patch)
             }
         }
 
+        /// <summary>
+        ///   Computes the final population of species, by patch.
+        /// </summary>
+        /// <remarks>
+        ///   <para>
+        ///     TODO: Currently no enforced correlation with actual changes applied.
```

Comment:
What's this TODO about?

Reply:
Wouldn't it then be better to put this data for example in a separate object? Perhaps internal to the extinction step if this data has no actual relation to the results.